testing/ef-tests/src/macros/fork_choice.rs

```diff
@@ -0,0 +1,256 @@
+#[macro_export]
+macro_rules! test_fork_choice {
+    ($path:ident) => {
+        paste::paste! {
+            #[cfg(test)]
+            #[allow(non_snake_case)]
+            mod [<tests_ $path>] {
+                use std::fs;
+
+                use alloy_primitives::{hex, map::HashMap, B256, hex::FromHex};
+                use ream_consensus::{
+                    attestation::Attestation,
+                    attester_slashing::AttesterSlashing,
+                    checkpoint::Checkpoint,
+                    deneb::{
+                        beacon_block::{BeaconBlock, SignedBeaconBlock},
+                        beacon_state::BeaconState,
+                    },
+                    execution_engine::{mock_engine::MockExecutionEngine, rpc_types::get_blobs::{Blob, BlobsAndProofV1}}, polynomial_commitments::kzg_proof::KZGProof,
+                };
+                use ream_fork_choice::{
+                    handlers::{on_attestation, on_attester_slashing, on_block, on_tick},
+                    store::{get_forkchoice_store, Store},
+                };
+                use rstest::rstest;
+                use serde::Deserialize;
+                pub const MAX_BLOB_COMMITMENTS_PER_BLOCK: u64 = 4096;
```

Comment:
```suggestion
```

Can you move this const to one of the other crates maybe forkchoice or consensus? up to you. In generate you shouldn't mix constants with imports like this again. Constants should always be defined below imports seperated by a new line